vllm/platforms/interface.py

```diff
@@ -27,16 +41,50 @@ def is_tpu(self) -> bool:
     def is_cpu(self) -> bool:
         return self._enum == PlatformEnum.CPU
 
-    @staticmethod
-    def get_device_capability(device_id: int = 0) -> Optional[Tuple[int, int]]:
+    def is_cuda_alike(self) -> bool:
+        """Stateless version of :func:`torch.cuda.is_available`."""
+        return self._enum in (PlatformEnum.CUDA, PlatformEnum.ROCM)
+
+    @classmethod
+    def get_device_capability(
+        cls,
+        device_id: int = 0,
+    ) -> Optional[DeviceCapability]:
+        """Stateless version of :func:`torch.cuda.get_device_capability`."""
         return None
 
-    @staticmethod
-    def get_device_name(device_id: int = 0) -> str:
+    @classmethod
+    def has_device_capability(
+        cls,
+        capability: Union[Tuple[int, int], int],
+        device_id: int = 0,
+    ) -> bool:
+        """
+        Test whether this platform is compatible with a device capability.
+
+        The ``capability`` argument can either be:
+
+        - A tuple ``(major, minor)``.
+        - A two-digit integer ``<major><minor>``.
+        """
+        current_capability = cls.get_device_capability(device_id=device_id)
+        if current_capability is None:
+            return False
+
+        if isinstance(capability, tuple):
+            return current_capability >= capability
+
+        if not 10 <= capability < 100:
+            raise ValueError("`capability` must be a two-digit integer.")
```

Comment:
final nit: this check is not necessary. the major can go beyond `9` in the future. let's don't hardcode this.

Reply:
Can we assume that the minor version will never exceed `9` in this case?

Reply:
Otherwise it may be complicated when both major and minor versions can have more than one digit.

Reply:
> Can we assume that the minor version will never exceed 9 in this case?

yes